app/app.go

```diff
@@ -394,8 +394,10 @@ func NewEthermintApp(
 		appCodec, app.GetSubspace(feemarkettypes.ModuleName), keys[feemarkettypes.StoreKey], tkeys[feemarkettypes.TransientKey],
 	)
 
+	// Create EVM keeper with a new field authority which is defaulted to the
```

Comment:
Avoid wording like `new field` in comments, unless you're writing a migration. The comments should describe the current functionality. 

```
	// Set authority to x/gov module account to only expect the module account to update params
```